wft4galaxy/app/runner.py

```diff
@@ -33,10 +40,14 @@ def _make_parser():
     parser.add_argument('--disable-cleanup', help='Disable cleanup', action='store_true', default=None)
 
     parser.add_argument('--output-format', choices=OutputFormat, help='Choose output type', default=OutputFormat.text)
-
     parser.add_argument('--xunit-file', default=None, metavar="FILE_PATH",
                         help='Set the path of the xUnit report file (absolute or relative to the output folder)')
     parser.add_argument('-o', '--output', dest="output_folder", metavar="PATH", help='Path of the output folder')
+
+    parser.add_argument('--max-retries', type=_check_positive, help='Max number of retries', default=None)
+    parser.add_argument('--retry-delay', type=_check_positive, help='Delay between retries', default=None)
```

Comment:
"Delay between retries in seconds" would read better.

Reply:
Fixed by https://github.com/phnmnl/wft4galaxy/pull/41/commits/7838cd970fa2072ceb3a39048a6e9d9e1d12383a